library/src/main/java/com/bumptech/glide/load/resource/bitmap/TransformationUtils.java

```diff
@@ -181,11 +224,11 @@ public static Bitmap centerInside(@NonNull BitmapPool pool, @NonNull Bitmap inBi
 
   /**
    * Sets the alpha of the Bitmap we're going to re-use to the alpha of the Bitmap we're going to
-   * transform. This keeps {@link android.graphics.Bitmap#hasAlpha()}} consistent before and after
+   * transform. This keeps {@link Bitmap#hasAlpha()}} consistent before and after
```

Comment:
There should be one included: https://github.com/bumptech/glide/tree/master/.idea

The code formatter of IDEA is not perfect, many times it has its own unconfigurable rules. Most noticed during refactors when static imports and inner class imports are freely changed.

What I usually do is do all the coding, use the formatter, and then while committing and reviewing my diff locally I just revert the code blocks I didn't specifically change. This prevents those weird diffs like you have after your second commit with the space alignments. They're not part of this PR and just confusing if later the commit diff is reviewed. I suggest you revert those.